website/docs/r/cdn_frontdoor_custom_domain.html.markdown

```diff
@@ -0,0 +1,135 @@
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-cdn-frontdoor"
+  location = "West Europe"
+}
+
+resource "azurerm_dns_zone" "example" {
+  name                = "afdx-terraform.azfdtest.xyz "
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_cdn_frontdoor_profile" "example" {
+  name                = "example-profile"
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_cdn_frontdoor_custom_domain" "example" {
+  name                     = "example-customDomain"
+  cdn_frontdoor_profile_id = azurerm_cdn_frontdoor_profile.example.id
+  dns_zone_id              = azurerm_dns_zone.example.id
+  host_name                = "contoso.com"
+
+  tls {
+    certificate_type    = "ManagedCertificate"
+    minimum_tls_version = "TLS12"
+  }
+}
+```
+## Example DNS Auth TXT Record Usage
+
+```hcl
+resource "azurerm_dns_txt_record" "example" {
+  name                = join(".", ["_dnsauth", "contoso"])
+  zone_name           = azurerm_dns_zone.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  ttl                 = 3600
+
+  record {
+    value = azurerm_cdn_frontdoor_custom_domain.example.validation_token
+  }
+}
+```
+
+## Example CNAME Record Usage
+
+!>**IMPORTANT:** You **must** include the `depends_on` meta-argument which references both the `azurerm_cdn_frontdoor_route` and the `azurerm_cdn_frontdoor_security_policy` that are associated with your Custom Domain.
+
+```hcl
+resource "azurerm_dns_cname_record" "example" {
+  depends_on = [azurerm_cdn_frontdoor_route.example, azurerm_cdn_frontdoor_security_policy.example]
+
+  name                = "contoso"
+  zone_name           = azurerm_dns_zone.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  ttl                 = 3600
+  record              = azurerm_cdn_frontdoor_endpoint.example.host_name
+}
+```
+
+## Arguments Reference
+
+* `name` - (Required) The name which should be used for this Frontdoor Custom Domain. Possible values must be between 2 and 260 characters in length, must begin with a letter or number, end with a letter or number and contain only letters, numbers and hyphens. Changing this forces a new Frontdoor Custom Domain to be created.
+
+* `cdn_frontdoor_profile_id` - (Required) The ID of the Frontdoor Profile. Changing this forces a new Frontdoor Profile to be created.
+
+* `host_name` - (Required) The host name of the domain. Changing this forces a new Frontdoor Custom Domain to be created.
+
+* `dns_zone_id` - (Optional) The Resource ID of the DNS Zone that is to be used for the Frontdoor Custom Domain.
+
+* `pre_validated_cdn_frontdoor_custom_domain_id` - (Optional) The resource ID of the pre-validated Frontdoor Custom Domain. This domain type is used when you wish to onboard a validated Azure service domain, and then configure the Azure service behind an Azure Front Door.
```

Comment:
```suggestion
* `pre_validated_cdn_frontdoor_custom_domain_id` - (Optional) The ID of the pre-validated CDN FrontDoor Custom Domain. This domain type is used when you wish to onboard a validated Azure service domain, and then configure the Azure service behind an Azure Front Door.
```

Reply:
This feature is being put temporarily on hold... I will skip this for not... 🙂